### PR TITLE
PMM-7072 client docker

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/setting-up/client/docker.md [hunk 1]
```diff
@@ -1,96 +1,57 @@
...
 
-To get help:
+| Variable                        | Description   |
+| ------------------------------- | ------------- |
+| `PMM_AGENT_SERVER_ADDRESS`      | The PMM Server hostname and port number.
+| `PMM_AGENT_SERVER_USERNAME`     | The PMM Server user name.
+| `PMM_AGENT_SERVER_PASSWORD`     | The PMM Server user’s password.
+| `PMM_AGENT_SERVER_INSECURE_TLS` | If true (1), use insecure TLS. Otherwise, do not.
+| `PMM_AGENT_SETUP`               | If true (1), run `pmm-agent setup`. Default: false (0).
+| `PMM_AGENT_CONFIG_FILE`         | The PMM Agent configuration file.
```
Comment (rnovikovP) [inline docs/setting-up/client/docker.md:54]: If you  run 
docker run --rm perconalab/pmm-client:2 --help
You can see the full list of parameters/variables available.  
Maybe we should include all of them?

File: docs/setting-up/client/docker.md [hunk 2]
```diff
@@ -1,96 +1,57 @@
...
+            -e PMM_AGENT_SERVER_PASSWORD=admin \
+            -e PMM_AGENT_SERVER_INSECURE_TLS=1 \
+            -e PMM_AGENT_SETUP=1 \
+            -e PMM_AGENT_CONFIG_FILE=pmm-agent.yml \
+            --volumes-from pmm-client-data perconalab/pmm-client:2
 
-`PMM_AGENT_SETUP`
-: If true (1), run `pmm-agent setup`. Default: false (0).
+!!! alert alert-success "Tip"
+    To get help: `docker run --rm perconalab/pmm-client:2 --help`
```
Comment (rnovikovP) [inline docs/setting-up/client/docker.md:43]:
```suggestion
    To get help: `docker run --rm percona/pmm-client:2 --help`
```

File: docs/setting-up/client/docker.md [hunk 3]
```diff
@@ -1,96 +1,57 @@
...
-`PMM_AGENT_SERVER_INSECURE_TLS`
-: If true (1), use insecure TLS. Otherwise, do not.
+        docker run --rm \
+            -e PMM_AGENT_SERVER_ADDRESS=PMMServer:443 \
+            -e PMM_AGENT_SERVER_USERNAME=admin \
+            -e PMM_AGENT_SERVER_PASSWORD=admin \
+            -e PMM_AGENT_SERVER_INSECURE_TLS=1 \
+            -e PMM_AGENT_SETUP=1 \
+            -e PMM_AGENT_CONFIG_FILE=pmm-agent.yml \
+            --volumes-from pmm-client-data perconalab/pmm-client:2
```
Comment (rnovikovP) [inline docs/setting-up/client/docker.md:40]:
```suggestion
            --volumes-from pmm-client-data percona/pmm-client:2
```

**Review by rnovikovP — CHANGES_REQUESTED** [review]
see typos re  perconalab

File: docs/setting-up/client/docker.md (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/setting-up/client/docker.md:54]: Good idea, I'll try.

Comment (PaulJacobs-percona) [reply docs/setting-up/client/docker.md:56]: The command lists PMM Agent env vars. These are already described in the new pmm-agent man page. So I shall consolidate the env vars there and mention here that PMM agent envvars can be used.

File: docs/setting-up/client/docker.md (hunk 3 above)
Comment (PaulJacobs-percona) [reply docs/setting-up/client/docker.md]: Already spotted and changed

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**